src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+
+class LoginForm extends Component {
+  render() {
+    return (
+      <View>
+        <Text> LoginForm </Text>
```

Comment:
Is there a reason for the multiple <Text> duplicates?

Reply:
Nope just so I can see a lot of text on the screen when I navigated to another component.